fuel-vm/src/util.rs

```diff
@@ -211,13 +211,30 @@ pub mod test_helpers {
             self.block_height
         }
 
+        #[cfg(any(fuzzing, feature = "test-helpers"))]
+        pub fn start_script_bytes(
```

Comment:
```suggestion
        pub fn start_script_bytes(
```

Reply:
Done in https://github.com/FuelLabs/fuel-vm/pull/733/commits/86f73591b00cdc297ef4905c4ebd4b6ee0dc7d94